package.json

```diff
@@ -14,28 +14,34 @@
   "license": "MIT",
   "devDependencies": {
     "@twilio-labs/serverless-api": "^1.1.0",
-    "airtable": "^0.8.1",
+    "airtable": "^0.11.0",
```

Comment:
can you update all of these versions in their respective templates as well? Since the version bumps here only apply to local template development not to the customer's templates